src/cmi_docx/comment.py

```diff
@@ -135,3 +137,249 @@ def add_comment(
     comment_reference.append(comment_reference_run)
 
     elements[0].append(comment_reference)
+
+
+@dataclass
```

Comment:
Minor styling: just to remain consistent with the rest of the codebase, can you import the module, not the function?